code/modules/economy/holopay.dm

```diff
@@ -0,0 +1,249 @@
+/// Max range at which the hologram can be projected before it deletes
+#define MAX_HOLO_RANGE 4
+/// Minimum forced fee for holopay stations. Registers as "pay what you want."
+#define MIN_FEE 0
+/// Maximum forced fee. It's unlikely for a user to encounter this type of money, much less pay it willingly.
+#define MAX_FEE 5000
```

Comment:
Same can to some degree be said of these, though it might be funny to make them vars on the machine